koalabot.py

```diff
@@ -58,17 +58,24 @@
 
 def is_owner(ctx):
     """
-    A command used to check if the user of a command is the owner, or the testing bot
+    A command used to check if the user of a command is the owner, or the testing bot.
+    The command also allows Senior Devs of KoalaBot to use owner only commands (as given by Admin role in the dev portal)
     e.g. @commands.check(koalabot.is_owner)
     :param ctx: The context of the message
     :return: True if owner or test, False otherwise
     """
     if is_dm_channel(ctx):
         return False
     elif BOT_OWNER is not None:
-        return ctx.author.id == int(BOT_OWNER) or is_dpytest
+        if ctx.author.id == int(BOT_OWNER):
+            return True
+        else:
+            return ctx.message.author.name in ADMINS or is_dpytest
     else:
-        return bot.is_owner(ctx.author) or is_dpytest
+        if ctx.message.author.name in ADMINS:
+            return True
+        else:
+            return bot.is_owner(ctx.author) or is_dpytest

```

Comment:
This doesn't look quite right, and the id should be used instead of author name (in case anyone changes account names). This will need modification based on changing to just using BOT_OWNER. (e.g. split by `,` then see if author.id is in that list)